src/browser/main.js

```diff
@@ -11,6 +11,9 @@ import {Provider} from 'react-redux';
 // TODO: Add app storage example.
 // import storage from 'redux-storage';
 
+// Enabling ES7 `async/await` in browser:
+if (process.env.IS_BROWSER) require('babel-core/polyfill');
```

Comment:
NP, you're welcome. 
I'll change it to only require `regenerator/runtime` instead of complete `polyfill`, right.